src/include/esp-idf/bindings.h

```diff
@@ -202,3 +201,5 @@
 #endif
 
 #endif
+
+#include "mdns.h"
```

Comment:
`mdns` is [a separate component in ESP-IDF](https://github.com/espressif/esp-idf/tree/master/components/mdns), so you can't just unconditionally include its header, or else e.g. CMake-driven builds will start to fail. What you need to do is as follows:
* Declare a new `mdns` component [here](https://github.com/esp-rs/esp-idf-sys/blob/master/build/common.rs#L33).
* Conditionalize the inclusion of the `mdns` header similarly to [this one](https://github.com/esp-rs/esp-idf-sys/blob/master/src/include/esp-idf/bindings.h#L58)